docs/features/remote-payments.md

```diff
@@ -49,7 +49,8 @@ Name | Return | Default | Description
 `needs_setup()` | boolean | `false` | Used to determine if the gateway still requires setup in order to be used.
 `get_required_settings_keys()` | array | `[]` | An array of keys for fields required to properly configure the gateway.  The keys must match those of already registered form fields in the payment gateway.
 `get_oauth_connection_url()` | string | `null` | The oAuth connection URL to be used to quickly connect a payment gateway provider.  If provided, this will be used in place of required setting fields.
-`get_post_install_script()` | string | `null` | An abolute URL to a script that will be dynamically loaded into a page after a gateway has been installed.  This is primarily used for `SlotFill`, but can allow any script to be added to assist in payment gateway setup.
+`get_post_install_scripts()` | array | `[]` | An abolute URL to a script that will be dynamically loaded into a page after a gateway has been installed.  This is primarily used for `SlotFill`, but can allow any script to be added to assist in payment gateway setup.
```

Comment:
```suggestion
`get_post_install_scripts()` | array | `[]` | An array of script URLs that will be dynamically loaded into a page after a gateway has been installed.  This is primarily used for `SlotFill`, but can allow any script to be added to assist in payment gateway setup.
```

There's a typo and maybe plural scripts?

Reply:
Great catch, @ilyasfoo!  I was way too hasty updating this.  This array was actually updated to script handles instead of URLs.  Revised this description to meet the current setup.